versionist.conf.js

```diff
@@ -9,13 +8,22 @@ module.exports = {
   incrementVersion: 'semver',
   updateVersion: 'npm',
 
+  // Only include a commit when there is a footer of 'Change-Type'.
+  // Ensures commits which do not up versions are not included.
+  // It does mean that commit messages without a relevant footer will not be included in the CHANGELOG.
+  includeCommitWhen: (commit) => {
+    if (commit.footer['Change-Type']) {
+      return true;
+    }
+    return false;
+  },
+
   // Determine the type from 'Change-Type:' tag.
-  // Should no explicit change type be made, then patch is assumed.
+  // Should no explicit change type be made, then no changes are assumed.
   getIncrementLevelFromCommit: (commit) => {
     if (commitType = commit.footer['Change-Type']) {
```

Comment:
After the changes to `includeCommitWhen`, you're ensured to have a change type here, so you can just `return commit.footer['Change-Type']`.